backend/ideas/utility/idea.py

```diff
@@ -125,30 +125,22 @@ def idea_translations(
             idea_translation.save()
 
 
-def get_idea_name(idea: Idea, language_code: str) -> str:
-    if (
-        language_code != idea.language.language_code
-        and idea.translate_idea.filter(language__language_code=language_code).exists()
-    ):
-        return (
-            idea.translate_idea.filter(language__language_code=language_code)
-            .first()
-            .name_translation
-        )
+def get_translation(idea: Idea, language_code: str) -> Optional[IdeaTranslation]:
+    if idea.language and language_code != idea.language.language_code:
+        return idea.translate_idea.filter(language__language_code=language_code).first()
+    return None
```

Comment:
We should definitely make these generic all over the code base but for this PR it's okay to do it like this.